pkg/venafi/tpp/connector.go

```diff
@@ -1253,6 +1286,8 @@ func (c *Connector) getIdentityMatching(identities []policy.IdentityEntry, ident
 	} else {
 		return nil, fmt.Errorf("it was not possible to find the user %s", identityName)
 	}
+
+	return &(identities[0]), nil
```

Comment:
Given the `if condition `above of this line, this will never be executed.

Reply:
Oops, I forgot to undo this change before committing. I'll undo it now.